### PR TITLE
Gradle

### PR DESCRIPTION
**Pull request by jburel**
To test: 
* Run ``gradle build``
* Check that a zip is generated under ``build/distributions``
* Select the zip, unzip
* Add the folder and subfolder to the Matlab path
* Check that you can load OMERO and connect to the server
You can try a "full workflow" using scripts from https://github.com/ome/training-scripts/tree/master/practical/matlab

Note that currently all the jars included under libs are added to the java classpath.
This could be filtered.
B-F ones are now added, this should allow us to include the approach followed by @kouichi-c-nakamura 

The zip will be available on GitHub when a tag is pushed

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    }
+    from file("src/ice.config")
+    into "${project.buildDir}/dist"
+
+}
+
+task packageDistribution(type: Zip) {
+    archiveFileName = "OMERO.matlab-"+version+".zip"
+
+    from "${project.buildDir}/dist"
```
Comment (rgozim) [inline build.gradle]: `from copyFiles`

**Comment by rgozim** [comment]
Tested with simple_frap.m and it works fine 👍

**Comment by pwalczysko** [comment]
Works for me fine with Matlab R2018b.

- connect to outreach
- retrieve all the images in a dataset, open them
- create a csv

**Comment by jburel** [comment]
see https://github.com/ome/omero-matlab/issues/11

merging for m1

**Pull request merged**